CoverageProfiler/CoverageProfiler.wdl

```diff
@@ -0,0 +1,223 @@
+version 1.0
+
+workflow coverageProfile {
+    input {
+        String sampleName
+        String coverageTool = "Samtools"
+        File alignedBam
+        File alignedBamIndex
+        File referenceFasta
+        File referenceDict
+        File referenceFai
+        File intervals
+        File? interval_GCcontent_track
+        Int MinBaseQuality = 20
+        Int MinMappingQuality = 20
+        Boolean visualise_coverage = false
+    }
+    if (coverageTool =="Samtools") {
```

Comment:
We should consider adding a linter so that we can fix things like whitespace without going through a review.

Reply:
Interesting. I am surprised Winstanley WDL didn't suggest that change. There is a WDL lint we could incorporate. But, it is not as convenient as a IDE plugin. 
https://github.com/dnanexus/wdlTools/blob/develop/doc/Commands/Lint.md